docs/rest_api_guide/06_GrafanaExamples.rst

```diff
@@ -0,0 +1,153 @@
+..
+.. Images
+.. |GrafanaAsset| image:: ../images/Grafana_asset.jpg
+.. |GrafanaPing| image:: ../images/Grafana_ping.jpg
+.. |GrafanaReading| image:: ../images/Grafana_reading.jpg
+.. |GrafanaStatistics| image:: ../images/Grafana_statistics.jpg
+.. |GrafanaTimestamp| image:: ../images/Grafana_Timestamp.jpg
+
+.. Links
+.. |grafana| raw:: html
+
+   < a href="https://grafana.com" target="_blank">Grafana</a>
+
+Grafana Examples
+================
+
+The REST API of Fledge provides a way to integrate other applications with Fledge, these applications can control Fledge or that may be used to monitor the operation of Fledge itself or to visualize the data held within a Fledge instance. One such tool is |grafana|. Here we will show some simple examples of how the Fledge REST API can be used with Grafana and the Infinity data source plugin. This is intended to be a simple example, more complex systems can be built using these tools.
+
+Show Fledge Status
+------------------
+
+Using the *GET /fledge/ping* endpoint we can retrieve information about the number of readings read, sent, purged etc. 
+
+.. code-block:: JSON
+
+	{
+	  "uptime": 13203,
+	  "dataRead": 2045868,
+	  "dataSent": 6700,
+	  "dataPurged": 1293723,
+	  "authenticationOptional": true,
+	  "serviceName": "Fledge",
+	  "hostName": "foglamp-18",
+	  "ipAddresses": [
+	    "192.168.0.172"
+	  ],
+	  "health": "green",
+	  "safeMode": false,
+	  "version": "1.9.2"
+	}
+
+We can use this URL as the query for a Grafana dashboard panel to retrieve the basic statistics. We then select the items we want to display as columns and set the type of these, in this case we have chosen the basic counters which are numeric value.
+
++---------------+
+| |GrafanaPing| |
```

Comment:
Note: As we have used screenshot with fledge occurrence. We need to change this for white-labelling.

Reply:
yes, a slight pain, but just something to remember